zkevm-circuits/src/circuit_tools/constraint_builder.rs

```diff
@@ -454,6 +458,24 @@ impl<F: Field, E: Expressable<F>> Expressable<F> for (E, E, E, E) {
     }
 }
 
+/// Enables column query without knowing the Column type
+pub trait Querier<F: Field> {
+    /// Query a column at a relative position
+    fn query(self, meta: &mut VirtualCells<F>, at: Rotation) -> Expression<F>;
+}
```

Comment:
Nice! Makes the API similar to what https://github.com/privacy-scaling-explorations/halo2/pull/154.

Reply:
Awesome, I left a comment, I think the `expr` method name is confusing and perhaps `cur` would be better for the method name.